HEFSConverter/DssEnsembleReader.cs

```diff
@@ -49,8 +73,12 @@ public ITimeSeriesOfEnsembleLocations ReadDataset(Watersheds watershed, DateTime
 
 			using (DSSIO.DSSReader dss = new DSSIO.DSSReader(dssPath))
 			{
-				DSSIO.DSSPathCollection dssPaths = dss.GetCondensedPathNames(); // sorted
-				if (dssPaths.Count == 0)
+				DSSIO.DSSPathCollection rawDssPaths = dss.GetCondensedPathNames(); // sorted
+                var dssPaths = rawDssPaths.OrderBy(a => a, new PathComparer()).ToArray(); // sorted
```

Comment:
Does this mean we are sorting twice? what will that do to performance?

Reply:
Perhaps we can stop the timer, during sorting to eliminate this concern, during experimentation?